.github/workflows/development-environment.yml

```diff
@@ -105,6 +106,24 @@ jobs:
           source .venv/bin/activate
           [[ $(python -V) == "Python $(cat .python-version)" ]]
 
+  tools-tests:
+    runs-on: ubuntu-20.04
+    timeout-minutes: 5
+    steps:
+      - uses: actions/checkout@v3
+      - uses: actions/setup-python@v3
```

Comment:
Oh, there's v3 now with `cache-dependency-path` stuff? That's pretty cool, you could f/u with a PR to upgrade to v3 and remove like, this stuff:

```
  cache-files-hash:
    description: 'A single hash for a set of files. Used for caching.'
    required: false
    default: ${{ hashFiles('requirements-*.txt', '!requirements-pre-commit.txt') }}
  pip-cache-version:
```

Reply:
ah yeah they just added this -- it works ~fairly well